dashboard/config/routes.rb

```diff
@@ -5,6 +5,7 @@ module OPS
   DASHBOARDAPI = 'dashboardapi' unless defined? DASHBOARDAPI
 end
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
Note we could instead exclude this entire file from this Robocop rule in the [configuration](http://rubocop.readthedocs.io/en/latest/configuration/#includingexcluding-files). I'm ok with either approach.